api-reference/20 Data Visualization Widgets/BaseChart/1 Configuration/tooltip/interactive.md

```diff
@@ -5,7 +5,7 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+#include dataviz-ref-tooltip-interactive
```

Comment:
It is better to use inheritance instead. Example:

https://raw.githubusercontent.com/DevExpress/devextreme-documentation/20_1/api-reference/30%20Data%20Layer/DataSource/3%20Methods/off(eventName).md